From penberg@cs.helsinki.fi  Wed Nov 12 13:42:49 2008
From: Pekka J Enberg <penberg@cs.helsinki.fi>
Date: Wed, 29 Oct 2008 20:11:49 +0200 (EET)
Subject: Staging: w35und: remove unused macros from common.h
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: Pavel Machek <pavel@suse.cz>
Message-ID: <Pine.LNX.4.64.0810292011140.16596@melkki.cs.Helsinki.FI>


From: Pekka Enberg <penberg@cs.helsinki.fi>

The macros are not used anywhere so remove them from driver code.

Acked-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/linux/common.h |   15 ---------------
 1 file changed, 15 deletions(-)

--- a/drivers/staging/winbond/linux/common.h
+++ b/drivers/staging/winbond/linux/common.h
@@ -33,21 +33,6 @@
 
 //#define DEBUG_ENABLED  1
 
-
-
-//===============================================================
-// Common type definition
-//===============================================================
-
-//===========================================
-#define IGNORE      2
-#define	SUCCESS     1
-#define	FAILURE     0
-
-
-#define STATUS_MEDIA_CONNECT 1
-#define STATUS_MEDIA_DISCONNECT 0
-
 //==================================================================================================
 // Common function definition
 //==================================================================================================
